.cirrus.yml

```diff
@@ -317,6 +384,29 @@ consistency_task:
     always: *runner_stats
 
 
+# Check that all included go modules from other sources match
+# what is expected in `vendor/modules.txt` vs `go.mod`.  Also
+# make sure that the generated bindings in pkg/bindings/...
+# are in sync with the code.
+consistency_aarch64_task:
+    name: "Test Code Consistency (aarch64)"
+    alias: consistency_aarch64
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: *is_pr
+    depends_on:
+        - build_aarch64
+    container: *smallcontainer
```

Comment:
get rid of this line (sorry I missed it).  You're using an `ec2_instance` for this task, not the container image.  But I don't think that explains fully the missing `Python.h`.  Still poking around.